spec.html

```diff
@@ -25345,6 +25345,9 @@ <h1>Number.isNaN ( _number_ )</h1>
 
       <emu-clause id="sec-number.issafeinteger">
         <h1>Number.isSafeInteger ( _number_ )</h1>
+        <emu-note>
+          <p>An integer is considered "safe" if and only if it can be exactly represented by a Number value that uniquely distinguishes it from all other integers.</p>
```

Comment:
```suggestion
          <p>An integer is considered "safe" if and only if it can be exactly represented by a Number value, and no other integer is also best represented by that Number.</p>
```